queue_processors/autoreduction_processor/tests/test_post_process_admin.py

```diff
@@ -111,6 +112,22 @@ def test_load_reduction_script(self):
         self.assertEqual(file_path, os.path.join(MISC['scripts_directory'] % 'WISH',
                                                  'reduce.py'))
 
+    @patch(DIR + '.autoreduction_logging_setup.logger.debug')
+    def test_reduction_started(self, mock_log):
+        """
+        Test: reduction started message has been sent and logged
+        When: called within reduce method
+        """
+        amq_client_mock = Mock()
+        ppa = PostProcessAdmin(self.message, amq_client_mock)
+        ppa.reduction_started()
+
+        mock_log.assert_called_with("Calling: %s\n%s",
```

Comment:
This is a nit pick, but I try to avoid `assert_called_with` in favour of `assert_called` for logging;

This means someone can update the logging without changing the tests so that we don't become tightly coupled. Since this only really needs to be human readable.

However the `assert_called_once_with` uses a set API so should use the `*_with` variant